src/Utils/Notifications.js

```diff
@@ -70,6 +70,11 @@ export const Success = ({ msg }) => {
 
 /** Error message handler */
 export const Error = ({ msg }) => {
+  // Handle session expiry
+  if (msg === "Authentication credentials were not provided.") {
+    notify("Session expired. Please Login again.", "error");
+    return;
+  }
```

Comment:
can you move to auth component or context, this seems to be odd place to have it

Reply:
I mean i will check if there is any place 

But whenever there is a error notification its gonna display 'Authentication credentials were not provided.' so I thought keeping it in the notification.js is the best thing